code/datums/smells/smell.dm

```diff
@@ -0,0 +1,88 @@
+#define SMELL_HORRIBLE 0
```

Comment:
The reason for this, is so that it's visible to files that come before this one. If we used BYOND `var/const/whatever` statements we wouldn't need to worry about it, but #defines are the way people here like things, for some reason...